<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewPacketMessageClass</class>
 <widget class="QMainWindow" name="NewPacketMessageClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NewPacketMessage</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>60</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>BBS</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cBBS">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>45</y>
      <width>640</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>60</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>From</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cFrom">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>640</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="cTo">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>95</y>
      <width>640</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>60</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Subject</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cSubject">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>640</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="cToHelper">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>95</y>
      <width>60</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>To...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cSend">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cPrint">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>5</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Print</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cSave">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>5</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cDelete">
    <property name="geometry">
     <rect>
      <x>195</x>
      <y>5</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cClose">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>5</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cUrgent">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>5</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Urgent</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="cMessageTypePrivate">
    <property name="geometry">
     <rect>
      <x>485</x>
      <y>5</y>
      <width>55</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Private</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="cMessageTypeBulletin">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>5</y>
      <width>55</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Bulletin</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="cMessageTypeNts">
    <property name="geometry">
     <rect>
      <x>595</x>
      <y>5</y>
      <width>55</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>NTS</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="cMessage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>155</y>
      <width>711</width>
      <height>416</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>733</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSend"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_a_File"/>
    <addaction name="actionOpen_a_Report"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionSave_As_No_Headers"/>
    <addaction name="separator"/>
    <addaction name="actionNTS_Message_Maker"/>
    <addaction name="separator"/>
    <addaction name="actionProcess_a_Report"/>
    <addaction name="actionClear_Remaining_Prompts"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="actionPrint_No_Headers"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_All"/>
    <addaction name="separator"/>
    <addaction name="actionDelete_Message"/>
   </widget>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="actionReply"/>
    <addaction name="actionForward"/>
    <addaction name="separator"/>
    <addaction name="actionSet_as_a_Private_Message"/>
    <addaction name="actionSet_as_a_NTS_Message"/>
    <addaction name="actionSet_as_a_Bulletin"/>
    <addaction name="separator"/>
    <addaction name="actionRequest_Delivery_Receipt"/>
    <addaction name="actionRequest_Read_Receipt"/>
    <addaction name="separator"/>
    <addaction name="actionKeep_ICS213_Local"/>
    <addaction name="actionOpen_in_Form"/>
    <addaction name="separator"/>
    <addaction name="actionLine_Draw_toggle"/>
    <addaction name="actionFont"/>
    <addaction name="actionReset_Font_default"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionReset_Form_to_Normal"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuActions"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSend">
   <property name="text">
    <string>Send</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionSave_As_No_Headers">
   <property name="text">
    <string>Save As, No Headers...</string>
   </property>
  </action>
  <action name="actionSave_All">
   <property name="text">
    <string>Save All...</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionOpen_a_File">
   <property name="text">
    <string>Open a FIle</string>
   </property>
  </action>
  <action name="actionMove_to_Folder">
   <property name="text">
    <string>Move to Folder...</string>
   </property>
  </action>
  <action name="actionOpen_a_Report">
   <property name="text">
    <string>Open a Report</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print...</string>
   </property>
  </action>
  <action name="actionPrint_No_Headers">
   <property name="text">
    <string>Print, No Headers...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionNTS_Message_Maker">
   <property name="text">
    <string>NTS Message Maker</string>
   </property>
  </action>
  <action name="actionProcess_a_Report">
   <property name="text">
    <string>Process a Report</string>
   </property>
  </action>
  <action name="actionClear_Remaining_Prompts">
   <property name="text">
    <string>Clear Remaining Prompts</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="actionDelete_Message">
   <property name="text">
    <string>Delete Message</string>
   </property>
  </action>
  <action name="actionReply">
   <property name="text">
    <string>Reply</string>
   </property>
  </action>
  <action name="actionForward">
   <property name="text">
    <string>Forward</string>
   </property>
  </action>
  <action name="actionSet_as_a_Private_Message">
   <property name="text">
    <string>Set as a Private Message</string>
   </property>
  </action>
  <action name="actionSet_as_a_NTS_Message">
   <property name="text">
    <string>Set as a NTS Message</string>
   </property>
  </action>
  <action name="actionSet_as_a_Bulletin">
   <property name="text">
    <string>Set as a Bulletin</string>
   </property>
  </action>
  <action name="actionRequest_Delivery_Receipt">
   <property name="text">
    <string>Request Delivery Receipt</string>
   </property>
  </action>
  <action name="actionRequest_Read_Receipt">
   <property name="text">
    <string>Request Read Receipt</string>
   </property>
  </action>
  <action name="actionKeep_ICS213_Local">
   <property name="text">
    <string>Keep ICS213 Local</string>
   </property>
  </action>
  <action name="actionOpen_in_Form">
   <property name="text">
    <string>Open in Form</string>
   </property>
  </action>
  <action name="actionLine_Draw_toggle">
   <property name="text">
    <string>Line Draw (toggle)</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="text">
    <string>Font...</string>
   </property>
  </action>
  <action name="actionReset_Font_default">
   <property name="text">
    <string>Reset Font (default)</string>
   </property>
  </action>
  <action name="actionReset_Form_to_Normal">
   <property name="text">
    <string>Reset Form to Normal</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
